README.md

```diff
@@ -338,19 +338,19 @@ Configuration of site-wide elements (`locale`, `title`, `description`, `url`, `l
 
 Three skins (default, light, and dark) are available to change the color palette of the theme.
 
-| `default.css` | `light.css` | `dark.css` |
+| `default.scss` | `light.scss` | `dark.scss` |
 | --- | --- | --- |
 | ![default skin](https://mmistakes.github.io/so-simple-theme/images/default-skin.png) | ![light skin](https://mmistakes.github.io/so-simple-theme/images/light-skin.png) | ![dark skin](https://mmistakes.github.io/so-simple-theme/images/dark-skin.png) |
 
 ```yaml
-skin: "/assets/css/skins/default.css"
-skin: "/assets/css/skins/light.css"
-skin: "/assets/css/skins/dark.css"
+skin: "/assets/css/skins/default.scss"
```

Comment:
I think the `skin` paths need to stay `.css` since those are the paths that will be added to the `head`, and HTML can’t read SCSS files.

Reply:
Oops! It's my fault. Yes, the `skin` paths need to stay in `.css`, but the link to `/assets/css/skins/default.css` needs to be modified to `/assets/css/skins/default.scss`.